src/lib/gssapi/mechglue/g_init_sec_context.c

```diff
@@ -121,6 +121,7 @@ OM_uint32 *		time_rec;
     gss_OID		selected_mech;
     gss_mechanism	mech;
     gss_cred_id_t	input_cred_handle;
+    OM_uint32		used_req_flags;
```

Comment:
I think Nico might have been saying that you can just reassign the parameter req_flags rather than introducing a new variable, and I agree.

But I think I see a bug here.  If an application calls gss_create_sec_context(), then gss_set_context_flags(), then repeatedly calls gss_init_sec_context() with zero flags, the req_flags from gss_set_context_flags() should be used.  I believe this code would only use those flags in the first call to gss_init_sec_context(), which would be a problem for an old mech.